tasks/gov.js

```diff
@@ -37,31 +46,212 @@ task("gov:setPricingConfig")
+        const isfork = args.isfork === undefined ? false : parseBool(args.isfork);
+        let fork;
+        if (isfork) {
+            fork = await setupGovernanceFork("installer");
+        }
+        const et = require("../test/lib/eTestLib");
+        const ctx = isfork ? fork.ctx: await et.getTaskCtx();
 
+        if (isfork) {
+            const admin = fork.admin;
+            await et.taskUtils.runTx(ctx.contracts.installer.connect(admin).installModules(addrs, await ctx.txOpts()));
+        } else {
+            await et.taskUtils.runTx(ctx.contracts.installer.installModules(addrs, await ctx.txOpts()));
+        }
+    });
+
+task("gov:setChainlinkPriceFeed")
+    .addPositionalParam("underlying")
+    .addPositionalParam("chainlinkAggregator")
+    .addOptionalParam("isfork", "Run on localhost, which is already forked from mainnet")
+    .setAction(async (args) => {
+        const isfork = args.isfork === undefined ? false : parseBool(args.isfork);
+        let fork;
+        if (isfork) {
+            fork = await setupGovernanceFork("governor");
+        }
+        const et = require("../test/lib/eTestLib");
+        const ctx = isfork ? fork.ctx: await et.getTaskCtx();
+
+        const underlying = isfork? args.underlying : await et.taskUtils.lookupToken(ctx, args.underlying);
+
+        if (isfork) {
+            const admin = fork.admin;
+            await et.taskUtils.runTx(ctx.contracts.governance.connect(admin).setChainlinkPriceFeed(underlying, args.chainlinkAggregator, await ctx.txOpts()));
+        } else {
+            await et.taskUtils.runTx(ctx.contracts.governance.setChainlinkPriceFeed(underlying.address, args.chainlinkAggregator, await ctx.txOpts()));
+        }
+    });
+
+task("gov:forkHealthScoreDiff", "Compare the health scores of accounts from a pair of JSON files")
+    .addPositionalParam("preGovActionFileName", "file name without .json suffix")
+    .addPositionalParam("postGovActionFileName", "file name without .json suffix")
+    .addPositionalParam("removeFilesAfterParsing", "true or false")
+    .setAction(async ({preGovActionFileName, postGovActionFileName, removeFilesAfterParsing}) => {
+        const fs = require('fs');
+        
+        const prePath = `${preGovActionFileName}.json`;
+        const postPath = `${postGovActionFileName}.json`;
+        
+        try {
+            const pre_gov_scores = require(`../${prePath}`);
+            const post_gov_scores = require(`../${postPath}`);
+
+            for (let account of Object.keys(pre_gov_scores)) {
+                if (
+                    pre_gov_scores[account].health > 1.15 &&
+                    post_gov_scores[account].health > 1 && 
+                    post_gov_scores[account].health <= 1.15
+                ) {
+                    console.log(`Account ${account} is at risk of liquidation due to governance action`);
+                    console.log(`Account Log: ${JSON.stringify(accountLog, null, 4)}`);
```

Comment:
what is `accountLog`? I think we should display here collateral and liability value